mozilla/addons-server
# Bump pkcs7_digest algorithm to sha256

Fixes: mozilla/addons/issues/14804

### Testing

1. Download a sample addon ([here](https://staticfil.es/webext-generator/))
2. Upload via devhub (unlisted is better due to no 24 block on approval)
3. Auto approve the addon via `.manage.py`
4. View the product listing page
5. Find the .zip file in storage
6. Upload the zip file to [xpidump](https://williamdurand.fr/xpidump/)

Expect:
- addon is signed with sha256

### Checklist

- [X] Add or update relevant [docs](https://github.com/mozilla/addons-server/blob/master/docs) reflecting the changes made.
